### ISSUE TITLE
Training accuracy problem

### ISSUE THREAD

**Issue by Sirajabudabe** [hamuchiwa/AutoRCCar#179]
Hi
I have a huge training problem , the accuracy is limited from 40% to 65% any advice please.
and how to use all the data for training not test , i tried every thing , could you explain the difference and use of each one (train-test) i really don't get it 
also when neural network loads the training data can it take more than one file and combine them together ??? 
thanks @hamuchiwa

**Comment by hamuchiwa** [comment]
duplicated
